clarity/evaluator/haaqi/haaqi.py

```diff
@@ -50,73 +60,90 @@ def haaqi_v1(
 
     # Auditory model for quality
     # Reference is no processing or NAL-R, impaired hearing
-    xenv, xbm, yenv, ybm, xsl, ysl, fsamp = eb.ear_model(x, fx, y, fy, hl, eq, level1)
+    (
+        reference_db,
+        xbmreference_basilar_membrane,
```

Comment:
I think xbmreference_basilar_membrane was meant to be just 'reference_basilar_membrane'

Reply:
Thanks for catching that @jonbarker68, corrected and I also noticed I'd not correctly updated the docstrings to use the new variable names.